StyleCop.Analyzers/StyleCop.Analyzers/OrderingRules/SA1212SA1213CodeFixProvider.cs

```diff
@@ -72,6 +79,12 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             return document.WithSyntaxRoot(syntaxRoot);
         }
 
+        private static bool HasLeadingBlankLines(AccessorDeclarationSyntax secondAccessor)
```

Comment:
:bulb: Param type can be generalized to `SyntaxNode`. Can also be extracted as an extension method.

The same can be applied to `GetLeadingTriviaWithoutLeadingBlankLines`.